{{ .ProjectSnake }}/.aspect/cli/config.yaml

```diff
@@ -22,7 +22,9 @@ configure:
 {{- if and .Computed.go .Scaffold.oci }}
     - .aspect/cli/go_image.star
 {{- end }}
-
+{{- if and .Computed.go .Scaffold.oci }}
```

Comment:
it's go, lets merge https://github.com/aspect-build/aspect-workflows-template/pull/175/files